### ISSUE TITLE
Listbox: should unselected single-select items include `aria-selected=false`?

### ISSUE THREAD

**Issue by craigkovatch** [w3c/aria-practices#1474]
The current practices doc says:

> * In a single-select listbox, the selected option has `aria-selected` set to `true`.
> * if the listbox supports multiple selection:
>   * The element with role `listbox` has `aria-multiselectable` set to `true`.
>   * All selected options have `aria-selected` set to `true`.
>   * All options that are not selected have `aria-selected` set to `false`.

I have interpreted this to imply that unselected options in a single-select listbox should have no value for `aria-selected`, vs multi-select listboxes where all options are expected to have a value, be it `true` or `false`.

Is this correct? Either way, it would be good to make this explicit in this document, as the current phrasing leaves this up to interpreteation.
